index.bs

```diff
@@ -4168,6 +4200,811 @@ opened</dfn> steps given |window|, |type| and |message|.
+   set.
+
+1. If |request|'s [=request/window=] is an [=environment settings object=]:
+
+  1. Let |environment settings| be |request|'s [=request/window=]
+
+  1. If there is a [=/browsing context=] whose [=active window=] is |environment
+     settings|' [=environment settings object/global object=], let |context id|
+     be the [=browsing context id=] for that context.
+
+1. Let |timestamp| be a [=time value=] representing the current date and time in UTC.
+
+1. Let |params| be [=/map=] matching the <code>network.BaseParameters</code>
+   production, with the <code>request</code> field set to |request data|, the
+   |navigation| field set to <code>navigation</code>, the <code>context</code>
+   field set to |context id|, the <code>timestamp</code> field set to
+   |timestamp|, the <code>redirectCount</code> field set to |redirect count|,
+   and the <code>isRedirect</code> field set to |is redirect|.
+
+1. Return (|related browsing contexts|, |params|)
+
+</div>
+
+#### The network.Cookie type #### {#type-network-Cookie}
+
+[=Remote end definition=] and [=local end definition=]
+
+<pre class="cddl local-cddl">
+network.Cookie = {
+    name: text,
+    ? value: text,
+    ? binaryValue: [ uint ]
+    domain: text,
+    path: text,
+    ? expires: js-uint,
+    size: js-uint,
+    httpOnly: bool,
+    secure: bool,
+    sameSite: "strict" / "lax" / "none",
+};
+</pre>
+
+The <code>network.Cookie</code> type represents a cookie.
+
+If the cookie value can be represented as a UTF-8 encoded string, the
+<code>value</code> field will be present. Otherwise the <code>binaryValue</code>
+field will be present and consist of an array of integers representing the bytes
+of the cookie value.
+
+<div algorithm>
+To <dfn>get a cookie</dfn> given |stored cookie|:
+
+Note: The definitions of |stored cookie|'s fields are from [[COOKIES]], except
+samesite-flag, which is from [[SAME-SITE-COOKIES]].
+
+1. Let |name| be the result of [=UTF-8 decode=] with |stored cookie|'s name field.
+
+1. Let |utf8 decoded value| be the result of [=UTF-8 decode without BOM or fail=]
+   with |stored cookie|'s value.
+
+1. If |utf8 decoded value| is failure, then:
+
+    1. Let |value| be null and |binary value| be an empty list.
+
+    1. For each |byte| in |stored cookie|'s value:
+
+      1. Append the [=byte/value=] of |byte| to |binary value|.
+
+   Otherwise: Let |value| be |utf8 decoded value| and |binary value| be null.
+
+1. Let |domain| be |stored cookie|'s domain field.
+
+1. Let |path| be |stored cookie|'s path field.
+
+1. Let |expires| be |stored cookie|'s expires field represented as a unix
+   timestamp, if set, or null otherwise.
+
+1. Let |size| be the byte length of the result of serializing |stored cookie|
+   as it would be represented in a <code>Cookie</code> header.
+
+1. Let |http only| be true if |stored cookie|'s http-only-flag is true, or false
+   otherwise.
+
+1. Let |secure| be true if |stored cookie|'s secure-only-flag is true, or false
```

Comment:
```suggestion
1. Let |secure| be true if |stored cookie|'s secure-flag is true, or false
```

secure-only-flag -> secure-flag?

Reply:
The RFC seems to use `secure-only-flag` for the data model.